src/components/alert/alert.styles.ts

```diff
@@ -20,60 +20,68 @@ export default css`
     border-top-width: calc(var(--sl-panel-border-width) * 3);
     border-radius: var(--sl-border-radius-medium);
     font-family: var(--sl-font-sans);
-    font-size: var(--sl-font-size-small);
+    font-size: var(--ts-font-sm);
```

Comment:
I'm not sure we should be embedding references to `--ts` vars directly into the library. We already have a layer that assigns the `--sl-` vars to their corresponding `--ts-` values; can we just use that? Otherwise we start introducing inconsistency.

Reply:
Ah, okay. I hadn't thought of that. I'll revert this & check other changes to make sure I've only used the `--sl-` var

Reply:
So there are some font related decs where I previously created a `--ts-` var and set a hard-coded value because there was no equivalent `--sl-` var & I didn't want to update the `--sl-` var. What do you think of leaving those as is? (`--ts-leading-5` is one example. It's set in `tokens.css` as `1.25rem`)